tests_v2/test_layout_save_open.py

```
@@ -0,0 +1,150 @@
+    opened_workspace = ""
+    layout = {
+        "direction": "horizontal",
+        "split": 0.4,
+        "children": [
+            {"pane": {"surfaces": [{"type": "terminal", "name": "Left"}]}},
+            {
+                "direction": "vertical",
+                "split": 0.65,
+                "children": [
+                    {"pane": {"surfaces": [{"type": "browser", "url": "https://example.com", "name": "Docs"}]}},
+                    {"pane": {"surfaces": [{"type": "terminal", "cwd": "/tmp", "name": "Bottom"}]}},
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)